src/prerequisites/tests/test_forms.py

```diff
@@ -9,9 +9,7 @@ class PrereqFormsTest(TenantTestCase):
     def generate_prereq_model_choice(): 
         """ A list of tuples: [(model, model_search_field), ...] for DAL widgets"""
         models = IsAPrereqMixin.all_registered_model_classes()
-        model_choices = [(model, model.dal_autocomplete_search_fields()) for model in models]
-        # print([(model.__name__, model.dal_autocomplete_search_fields()) for model in models])
-        return model_choices
+        return [model for model in models]
```

Comment:
What is this doing? models (from all_registered_model_classes()) is already a list.  This seems to be a list comprehension to generate an identical copy of itself?  If we can't just return `models` here, please comment why we need this. Thanks!

Reply:
I didn't wrote this tests, just fixed them to work with new patches.